src/completed.js

```diff
@@ -0,0 +1,10 @@
+const clearAllCompleted = (todos) => todos.filter((task) => !task.completed);
+const IsChecked = (todos, s, i) => {
+    if (s.checked) {
+        todos[i].completed = true;
+    } else {
+        todos[i].completed = false;
+    }
+    localStorage.setItem('todos', JSON.stringify(todos));
+  };
+export { IsChecked, clearAllCompleted };
```

Comment:
I think, instaed of exporting different functions and variables separately you can encapsulate them using class and you only need to export the class. This helps you to make your code simple and Clean. **Remeber KISS**